src/5.Functions/home.php

```diff
@@ -40,6 +40,9 @@ function handleClick() {
         }
 
         if (isset($_REQUEST['revert_object'])) {
+            if (!isset($_SESSION['theObject'])) {
```

Comment:
Decided that this is how I would handle the vague and confused as well as confusing instructions, since functions cannot be overloaded in PHP except for methods in classes, hence a signature with and without parameter will not work.